lte/gateway/c/oai/include/s1ap_types.h

```diff
@@ -79,6 +80,14 @@ enum s1_ue_state_s {
                         /// UE context Release Complete
 };
 
+typedef struct handover_state_s {
+  mme_ue_s1ap_id_t mme_ue_s1ap_id;
+  uint32_t source_enb_id;
+  uint32_t target_enb_id;
+  enb_ue_s1ap_id_t
+      target_enb_ue_s1ap_id : 24;  ///< Unique UE id over eNB (24 bits wide)
+} handover_state_t;
```

Comment:
To keep the state naming convention, can we update this struct name to `s1ap_handover_state`? And also for its protobuf message types as well